vaadin-checkbox-flow-integration-tests/pom.xml

```diff
@@ -71,6 +71,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-button-testbench</artifactId>
+            <version>${project.version}</version>
```

Comment:
Don't use `${project.version}` for external artifacts. Versions might get out of synch once new features are added.

Reply:
> can dataProvider be null with the new reference?
No, because the the reference is `final`. Good catch. Fixed.

> Don't use ${project.version}
Done.

Reply:
Oh, now I recall why this `null` check is there...this dataProvider ref is used within the constructor and some identifier logic coupled with it. So, it's normal to have `null` reference here at the checkbox group object creation time.